pkg/sql/executor_normal_stmt.go

```diff
@@ -35,6 +35,17 @@ func runNormalStmt(wr *pipe.Writer, slct string, db *database.DB) error {
 func isQuery(slct string) bool {
 	s := strings.ToUpper(strings.TrimSpace(slct))
 	has := strings.Contains
+	// remove comment lines
+	if strings.HasPrefix(s, "-- ") {
```

Comment:
Also need to support multi-line comment: `/** .... **/`, can merge this PR and update in the future.

Reply:
Sure.